kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/HostapdConfigWriter.java

```diff
@@ -316,11 +316,11 @@ private String updateInterfaceAndDriver(String interfaceName, String interfaceDr
     }
 
     private String updateChannels(WifiConfig wifiConfig, String fileAsString) throws KuraException {
-        if (wifiConfig.getChannels()[0] > 0 && wifiConfig.getChannels()[0] <= 14) {
+        if (wifiConfig.getChannels()[0] > 0 && wifiConfig.getChannels()[0] <= 200) {
```

Comment:
If as reported in the message 124 is the maximum value, why are we setting here 200?

Reply:
Yes. The limit is in fact 165. I will also change the message.